sample-etc_ddclient.conf

```
@@ -210,8 +210,8 @@ ssl=yes					# use ssl-support.  Works with
 #protocol=cloudflare,        \
 #zone=domain.tld,            \
 #ttl=1,                      \
-#login=your-login-email,     \
-#password=APIKey             \
+#login=your-login-email,     \ # if using global key
```

```suggestion
#login=your-login-email,     \ # Only needed when you want to use a global api key.
```
I am not a native speaker so if someone has an even better idea feel free to improve me. :)